code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3256,6 +3259,40 @@
 	required_temp = T0C + 88 //Mutagen is very hard to heat up, so I don't recommend making more than 10u of this at a time
 	result_amount = 1
 
+/datum/chemical_reaction/aminomicin
+	name = "Aminomicin"
+	id = AMINOMICIN
+	result = AMINOMICIN
+	required_reagents = list(CLONEXADONE = 1, MUTAGEN = 1, ENZYME = 1)
+	result_amount = 1
+
+/datum/chemical_reaction/synthmouse
+	name = "Synthmouse"
+	id = "SYNTHMOUSE"
```

Comment:
you think you do but you don't

Reply:
Megathonk

Reply:
![image](https://user-images.githubusercontent.com/6307265/48135754-1469fa80-e29e-11e8-9e87-3b65a5c6b3ad.png)
you should drop the `""`